### ISSUE TITLE
cache_tool plugin can't find phar

### ISSUE THREAD

**Issue by foxycode** [automatephp/automate#35]
When added cachetools config section:
```yaml
plugins:
  cache_tool:
    version: 5.0.0
    opcache: true
```

I get following error:
```bash
php cachetool.phar opcache:reset --fcgi

 [ERROR] Could not open input file: cachetool.phar
```

Looks like I need to download cachetool manually, but in that case, what is `version` config option good for? Wouldn't be better to add `path` option? Does Automate look for `cachetool` executable in `$PATH`?

Also, what if my PHP isn't fcgi? This should be configurable too.

**Comment by julienj** [comment]
Hello,

Automate will download cachetools.
Can you try the following command from the server to see if there is not a network problem :
curl -sO  https://gordalina.github.io/cachetool/downloads/cachetool-5.0.0.phar 

Currently only fcgi is support but you can just use cache tools without the plugin

```
post_deploy:
    - "php path/to/cachetool.phar ..."
      
```

**Comment by foxycode** [comment]
curl downloads phar ok. What can I do to give you more data?

**Comment by julienj** [comment]
I just fixed the problem
can you test on the master?
if it's good i release a new version

**Comment by foxycode** [comment]
@julienj How do I download or generate phar from master?

**Comment by julienj** [comment]
We don't need to build the phar to test

`php bin/automate deploy ‹platform› -c [path_of_config_file]`

**Comment by julienj** [comment]
fix in 3.1
